src/vue/modules/verification/general-form/store/index.js

```diff
@@ -0,0 +1,133 @@
+import { DOCUMENT_TYPES } from '@/js/const/document-types.const'
+import { types } from './types'
+
+const state = {
+  formData: {
+    personal: {
+      firstName: '',
+      lastName: '',
+      dateOfBirth: '',
+    },
+    address: {
+      line1: '',
+      line2: '',
+      city: '',
+      state: '',
+      country: '', // // { code, translation }
+      postalCode: '',
+    },
+    documents: {
+      idDocumentType: {}, // { value, translationId }
+
+      avatar: null,
+      selfie: null,
+      idDocumentFace: null,
+      idDocumentBack: null,
+    },
+  },
```

Comment:
let’s consider making the formData flatten, also you should not use `Data` in variable names